src/test/resources/hudson/tasks/junit/JENKINS-41134/TestSuite_second.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+    <testsuite name="TestSuite_second" tests="1" errors="0" failures="1" skip="0" timestamp="2012-04-05T10:50:00">

```

Comment:
So the interesting thing with the Jenkins results is that the test suite/names are identical - so allthough this is a good test, it's not reproducing the issue as observed